internal/datastore/crdb/crdb.go

```diff
@@ -213,7 +213,7 @@ func (cds *crdbDatastore) Revision(ctx context.Context) (datastore.Revision, err
 }
 
 func (cds *crdbDatastore) SyncRevision(ctx context.Context) (datastore.Revision, error) {
-	ctx, span := tracer.Start(ctx, "SyncRevision")
+	ctx, span := tracer.Start(datastore.SeparateContextWithTracing(ctx), "SyncRevision")
```

Comment:
this seemed like an oversight - any reason we wouldn't want a separate context here?